### PR TITLE
Jess homepage setup

### PR DESCRIPTION
**Pull request by jessicapalomo**

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -25,14 +27,15 @@ import './assets/css/animate.min.css';
...
-
+import App from './App.jsx';
 import AdminLayout from './layouts/Admin.jsx';
 
 ReactDOM.render(
   <BrowserRouter>
     <Switch>
-      <Route path="/admin" render={(props) => <AdminLayout {...props} />} />
-      <Redirect from="/" to="/admin/dashboard" />
+      <Route path="/client/src/App" render={(props) => <App {...props} />} />
```
Comment (leopolicastro) [inline client/src/index.js:36]: This path refers to the path to get to this component in the URL, not the file path of the component in your code.  Is the URL for this component `http://localhost:3000/client/src/App` ??

File: client/src/pages/Home.jsx [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+import { Link } from 'react-router-dom';
 
-const Home = () => {
+function Home() {
   return (
     <div>
       <Navigation />
       <h1>Home</h1>
+      <h3>I need help</h3>
+      <Button as={Link} to="/UserSignup">
```
Comment (leopolicastro) [inline client/src/pages/Home.jsx:12]: You have it as `UserSignup` here but as `usersignup` in your react routes.

**Review by leopolicastro — CHANGES_REQUESTED** [review]
Great job! Just some small changes/questions.

File: client/src/index.js (hunk 1 above)
Comment (jessicapalomo) [reply client/src/index.js:36]: It should just be /App but was yelling at me for somer reason, until I put that full path

Comment (jessicapalomo) [reply client/src/index.js:36]: this should be /App but was yelling at me until I put the file path

**Pull request merged**